multinode_installation/playbooks/install_packages_local.yml

```diff
@@ -25,8 +25,6 @@
     - name: Install vagrant.1.7.4
       apt: deb="/tmp/vagrant.1.7.4.deb"
       sudo: true
-    - name: Install vagrant-libvirt
-      shell: vagrant plugin install vagrant-libvirt
```

Comment:
reason why this did not install any packages: command must be run from the directory of the Vagrantfile